README.md

```diff
@@ -1 +1,148 @@
+defines a Dataset. A Dataset in turn consists of a collection of Publisher Data
+Files. A Publisher Data File simulates the impression viewing history of a
+single publisher. It is a CSV file, where each line is of the form
+
+```
+   vid, spend
+```
+
+where vid is a virtual user id, and spend is the amount that would have to be
+spent for this impression to be shown.
+
+A Data Design can be synthetic, or it can represent actual publisher data.
+Synthetic data designs can be generated using the program
+`data_generators/synthetic_data_generator.py`. Alternatively, if you have other
+data available in your organization, you can construct your own data design. An
+example data design can be found in the directory `examples/data_designs`.
+
+An Experimental Design specifies the models that will be evaluated and the
+parameters that will be used for evaluation. An Experimental Design is given by
+a Python class. An Experimental Design is subdivided into Experiments, which are
+further broken down into Trials. An Experiment consists of a collection of
+models and model parameters that are run against a single Dataset. A Trial
+consists of simulating the outcome of one model using one Dataset. For an
+example Experimental Design, see the directory `examples/experimental_designs`.
+
+Once a Data Design and an Experimental Design have been specified, the Evaluator
+can be used to run simulations of the experimental design versus the data
+design. The final output of the Evaluator is a DataFrame, containing one row per
+Trial.
+
+The results of an Evaluator run can then be analyzed in Colab. For some example
+colabs, see the `analyzers` directory.
+
+### Quickstart
+
+It is recommended to use a virtual environment with this project. If you already
+have one, you can skip to the next step. The quickest way to set up a virtual
+environment is by running:
+
+```
+python3 -m venv env
+source env/bin/activate
+```
+
+To install the requirements as well as the evaluation framework, simply run:
+
+```
+pip install -r requirements.txt
+python setup.py install
+```
+
+After these steps, the code and its dependencies will be installed as Python
+packages.
+
+### Example
+
+This section walks you through the steps to set up the evaluation framework and
+to run a sample evaluation using synthetic data.
+
+To start with, you should create a directory that you will use for working:
+
+```
+DIR=<some-path-that-will-contain-data-and-results>
+mkdir -p $DIR
+```
+
+The next command invokes the synthetic data generator, instructing it to
+generate some synthetic data sets:
+
+```
+python3 data_generators/synthetic_data_generator.py \
+  -nreps=10 \
+  -npublishers=2 \
+  -cost_model=fixed \
+  -frequencies=heterogeneous:3:5 \
+  -prices=0.05,0.06,0.07 \
+  -overlap=independent
+  -output=$DIR/data_design
+```
+
+In this example, we will use the sample experimental design in the file
+`examples/experimental_designs/goerg_gamma_poisson.py`. This compares Goerg's
```

Comment:
Maybe link the paper and use Georg's last name?

Reply:
Done